modules/ROOT/pages/test-flow-with-scheduler-cookbook.adoc

```diff
@@ -0,0 +1,60 @@
+= Testing a Flow with a Scheduler Source
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Assume that you want to test the following flow triggered by an Scheduler component, and that you want to validate that your flow is being triggered correctly:
+
+
+[source,xml]
+----
+<flow name="schedulerFlow">
+    <scheduler>
+        <scheduling-strategy >
+            <fixed-frequency frequency="10" timeUnit="SECONDS"/>
+        </scheduling-strategy>
+    </scheduler>
+    <set-payload value="#['test poll']"/>
+</flow>
+----
+
+. Enable Flow Sources. +
+By default, MUnit does not start any of the event sources (flow sources such as triggers or listeners). You must configure which flow sources must start for your tests.
```

Comment:
Why introduce the synonym "event sources" here if we are otherwise referring to them as "flow sources"?
Using synonyms runs the risk of confusing audiences, who might assume that different wordings (for one thing) equates to different meanings.

Reply:
It might be simpler to reword to:
```suggestion
Because MUnit does not start any of the flow sources (such as triggers or listeners), you must configure which flow sources start for your tests.
```